.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         - test $(python -c 'import sys; print("%d%d" % sys.version_info[0:2])') = 37
 
     # Full run of latest supported version, without xdist.
-    - env: TOXENV=py37
+    - env: TOXENV=py37  PYTEST_COVERAGE=1
```

Comment:
We do not need this, do we?
We have py37-xdist with coverage already (at least).

Reply:
I'd guess it was the rebase that fixed it.

We can revert this and this - after things stabilized a bit.. :)